things/service.go

```diff
@@ -549,6 +605,93 @@ func (svc service) Identify(ctx context.Context, key string) (string, error) {
 	return client.ID, nil
 }
 
+func (svc service) VerifyConnections(ctx context.Context, req *magistrala.VerifyConnectionsReq) (*magistrala.VerifyConnectionsRes, error) {
```

Comment:
It should not be the gRPC response, It should like `ListObjects` in auth service. 
The return type should be service type and it should be converted into gRPC type in gRPC layer.